readthedocs/projects/models.py

```diff
@@ -114,6 +113,16 @@ class Project(models.Model):
         help_text=_("Google Analytics Tracking ID (ex. UA-22345342-1). "
                     "This may slow down your page loads."))
 
+    # Sphinx specific build options.
```

Comment:
I think we should probably leave the "sphinx" naming off these. They should be global settings for the project, and respected where they need to be, but having "sphinx" in the name feels weird.